pytorch_lightning/loggers/wandb.py

```diff
@@ -46,12 +49,180 @@ class WandbLogger(LightningLoggerBase):
     r"""
     Log using `Weights and Biases <https://docs.wandb.ai/integrations/lightning>`_.
 
-    Install it with pip:
+    **Installation and set-up**
+
+    Install with pip:
 
     .. code-block:: bash
 
         pip install wandb
 
+    Create a `WandbLogger` instance:
```

Comment:
Would you be ok if we put this in a separate documentation page? IMO user guide does not belong into the API docs here. What is your opinion?

Reply:
So it was actually inspired by [NeptuneLogger](https://pytorch-lightning.readthedocs.io/en/latest/extensions/generated/pytorch_lightning.loggers.NeptuneLogger.html#neptunelogger).

The reason is that W&B gets a lot of questions on this integration where the answers are in wandb doc but not in PL doc and people always refer to one of these pages when asking their questions:
* [WandbLogger](https://pytorch-lightning.readthedocs.io/en/latest/extensions/generated/pytorch_lightning.loggers.WandbLogger.html#pytorch_lightning.loggers.WandbLogger) where I added basic info in this PR (and referred to W&B colab and doc for more details)
* [Loggers](https://pytorch-lightning.readthedocs.io/en/latest/common/loggers.html#weights-and-biases) where I actually was going to duplicate this same info

Reply:
ok, your point about people not finding the answers in the docs is valid. 
But still I want to push on the issue off having extensive user guide mixed up with api docs.
I would like to make a follow up to this PR:
- give each logger a dedicated page
- on the current logger page, only summarize the third-party capabilities in a quick teaser section, redirect to individual pages
- the dedicated logger pages will contain more detailed usage examples like you show here, separated from the api documentation

However, for the individual page we also want to keep it concise and avoid duplicating the whole wandb documentation. 

What do you think of my thoughts here.

Reply:
That's a great idea.
Let's move this content to a dedicated page in a follow-up PR then. Feel free to set up how you want it to be organized and ping me to create/move specific doc.

Reply:
Great idea @awaelchli. Definitely would bring more visibility to all loggers :)